<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUI Extractor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/icons.qrc">
    <normaloff>:/icons/application.png</normaloff>:/icons/application.png</iconset>
  </property>
  <property name="accessibleName">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTreeWidget" name="topWindows">
        <property name="indentation">
         <number>0</number>
        </property>
        <column>
         <property name="text">
          <string>Window name</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>8</width>
          <height>268</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="buttonReload">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Reload list of top-level windows</string>
          </property>
          <property name="icon">
           <iconset resource="resources/icons.qrc">
            <normaloff>:/icons/reload.png</normaloff>:/icons/reload.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonAnalyze">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Analyse selected window</string>
          </property>
          <property name="icon">
           <iconset resource="resources/icons.qrc">
            <normaloff>:/icons/analyse.png</normaloff>:/icons/analyse.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>8</width>
          <height>268</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QTreeWidget" name="elementTree">
        <column>
         <property name="text">
          <string>Element name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Type</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="logArea">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>120</height>
       </size>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Toolbar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionEnableHighlighting"/>
   <addaction name="actionSelectHighlightingColor"/>
   <addaction name="separator"/>
   <addaction name="actionExportXUL"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>913</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuApplication">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuExtract">
    <property name="title">
     <string>Analyse</string>
    </property>
    <addaction name="actionSelectedWindow"/>
    <addaction name="actionReloadWindowList"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuExport">
    <property name="title">
     <string>Export</string>
    </property>
    <addaction name="actionExportXUL"/>
   </widget>
   <addaction name="menuApplication"/>
   <addaction name="menuExtract"/>
   <addaction name="menuExport"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSelectedWindow">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Selected window</string>
   </property>
   <property name="toolTip">
    <string>Analyse selected window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionEnableHighlighting">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/icons.qrc">
     <normaloff>:/icons/highlight.png</normaloff>:/icons/highlight.png</iconset>
   </property>
   <property name="text">
    <string>Highlighting</string>
   </property>
   <property name="toolTip">
    <string>Enable highlighting selected windows and elements</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionSelectHighlightingColor">
   <property name="icon">
    <iconset resource="resources/icons.qrc">
     <normaloff>:/icons/color.png</normaloff>:/icons/color.png</iconset>
   </property>
   <property name="text">
    <string>Select color</string>
   </property>
   <property name="toolTip">
    <string>Color used when highlighting windows and elements</string>
   </property>
  </action>
  <action name="actionExportXUL">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/icons.qrc">
     <normaloff>:/icons/export.png</normaloff>:/icons/export.png</iconset>
   </property>
   <property name="text">
    <string>Save as XUL</string>
   </property>
   <property name="toolTip">
    <string>Export GUI description in XUL format</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionReloadWindowList">
   <property name="text">
    <string>Reload window list</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
